litedram/phy/ddr5/commands.py

```diff
@@ -175,7 +175,8 @@ class Command(Module):
         "MRR":           ["H L H L H MRA0 MRA1 MRA2 MRA3 MRA4 MRA5 MRA6 MRA7 V",
                           "L L V V V V V V V V CW V V V"],
         "MRW":           ["H L H L L MRA0 MRA1 MRA2 MRA3 MRA4 MRA5 MRA6 MRA7 V",
-                          "OP0 OP1 OP2 OP3 OP4 OP5 OP6 OP7 V V CW V V V"]
+                          "OP0 OP1 OP2 OP3 OP4 OP5 OP6 OP7 V V CW V V V"],
+        "DESELECT":      ["X X X X X X X X X X X X X X"]
```

Comment:
Given that deselect has all don't cares, we can handle it as previously, nothing should change there I think.

One thing to change though is the CS value, which is inverted, it is LOW in the first half of the transaction, and HIGH in the second, while for the `DESELECT` it is only HIGH